### ISSUE TITLE
Can you create WUA (CEMU compatible) archives with ZArchive?

### ISSUE THREAD

**Issue by jeffrey-lebowski** [Exzap/ZArchive#19]
I'm aware that you can losslessly extract WUA files created in CEMU back to RPX/RAW/code-content-meta format with ZArchive, but is the reverse possible, i.e. to create WUA files from RPX/RAW/code-content-meta files without CEMU? 

I read here that WUA files "are otherwise regular ZArchive files", so I was hoping that maybe yes, but when I try to run `zarchive.exe <game dir>`, I get the `Failed to create archive file` error.

The problem I have with doing this through CEMU is that, afaik, the only way to do it is manually, through GUI and therefore I cannot write a script for it...

Thanks for developing and providing support for this useful tool.

**Comment by Exzap** [comment]
Yes you can create .wua files which are readable by Cemu via `zarchive.exe`.
The readme on this repository has a short section on how the files need to be laid out for Cemu to recognize them. You can also see this layout when you extract an existing .wua.

The command line you provided is correct (it's just `zarchive.exe <input path>`). Not sure about the error you got, but I suspect it might be because of unicode characters in the path. The Windows zarchive.exe tool currently has issues with this.
